.github/workflows/cargo_publish.yml

```diff
@@ -0,0 +1,42 @@
+name: cargo_publish
```

Comment:
I tested version_bump.yml a lot, but I did not test this workflow because I didn't want to test crates. Please review it carefully. If this fails during a release, we can easily execute this stuff manually (I should do the first release with this)